odoo/addons/base/tests/test_ir_actions_report.py

```diff
@@ -0,0 +1,102 @@
+from odoo import Command
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestIrActionsReport(AccountTestInvoicingCommon):
+
+    def setup(self):
+        super().setup()
+        self.first_invoice = self.env['account.move'].create({
```

Comment:
you can't make a reference to account.move in base, you can install base without installing account so in this case we would run this test and it would fail because `self.env['account.move']` returns an error

so this test is not doable in base module. Also I don't understand much what it solves to put in in base, since it is in post_install mode anyway (so if we installed both based and account, it would be run after the installation of both anyway°